ambari/src/main/java/io/brooklyn/ambari/server/AmbariServer.java

```diff
@@ -58,7 +58,7 @@
     AttributeSensor<String> CLUSTER_STATE = Sensors.newStringSensor("ambari.server.clusterState");
 
     BasicAttributeSensorAndConfigKey.StringAttributeSensorAndConfigKey USERNAME =
-            new BasicAttributeSensorAndConfigKey.StringAttributeSensorAndConfigKey("ambari.server.http.user", "Ambari Server's http username", "admin");
+            new BasicAttributeSensorAndConfigKey.StringAttributeSensorAndConfigKey("ambari.server.http.user", "Ambari Server's http username. It is always admin", "admin");
 
```

Comment:
This would be helpful if we want to be able in future to configure new users. It's fast and easy to implement as it's very similar to the `updateUser` implementation, but I'm not sure if we want it. WDYT, is it a reason to keep this code?